tests/functional/spec/features/calculated_summary.spec.js

```diff
@@ -104,25 +105,22 @@ describe("Feature: Calculated Summary", () => {
       expect($(CurrencyTotalPlaybackPageWithFourth.calculatedSummaryAnswer()).getText()).to.contain("£40.71");
     });
 
-    it("Given I leave an answer empty, When i get to the currency summary, Then I should see no answer provided and new total", () => {
-      $(CurrencyTotalPlaybackPageWithFourth.fourthNumberAnswerEdit()).click();
-      $(FourthNumberBlockPage.fourthNumber()).setValue("");
-      $(FourthNumberBlockPage.submit()).click();
+    it("Given I leave an answer empty, When I get to the currency summary, Then I should see no answer provided and new total", () => {
+      $(CurrencyTotalPlaybackPageWithFourth.fourthAndAHalfNumberAnswerAlsoInTotalEdit()).click();
       $(FourthAndAHalfNumberBlockPage.fourthAndAHalfNumberAlsoInTotal()).setValue("");
       $(FourthAndAHalfNumberBlockPage.submit()).click();
-
       $(FifthNumberBlockPage.submit()).click();
       $(SixthNumberBlockPage.submit()).click();
 
       expect(browser.getUrl()).to.contain(CurrencyTotalPlaybackPageWithFourth.pageName);
       expect($(CurrencyTotalPlaybackPageWithFourth.calculatedSummaryTitle()).getText()).to.contain(
-        "We calculate the total of currency values entered to be £9.36. Is this correct?"
+        "We calculate the total of currency values entered to be £28.37. Is this correct?"
       );
-      expect($(CurrencyTotalPlaybackPageWithFourth.calculatedSummaryAnswer()).getText()).to.contain("£9.36");
-      expect($(CurrencyTotalPlaybackPageWithFourth.fourthNumberAnswer()).getText()).to.contain("No answer provided");
+      expect($(CurrencyTotalPlaybackPageWithFourth.calculatedSummaryAnswer()).getText()).to.contain("£28.37");
+      expect($(CurrencyTotalPlaybackPageWithFourth.fourthAndAHalfNumberAnswerAlsoInTotal()).getText()).to.contain("No answer provided");
     });
 
-    it("Given I skip the fourth page, When i get to the playback, Then I can should not see it in the total", () => {
+    it("Given I skip the fourth page, When I get to the playback, Then I can should not see it in the total", () => {
```

Comment:
Similar here it's not clear that this has been edited from a different value. Also why has it gone back to the third block just to step through it to get to the one it wants to change?

Reply:
> Similar here it's not clear that this has been edited from a different value.

Not sure I follow what you mean.

> Also why has it gone back to the third block just to step through it to get to the one it wants to change?

It's the first question from the calculated summary it can go to before the `skip yes / no` question. The blocks in between aren't on the first calculated summary, they belong to unit calculated summary, etc.

Reply:
The first command is thirdNumberAnswerEdit, the description fails to mention it must have had a value already and is being edited. Can it not go to skip-fourth-block? But I will let it go because it wasn't the best to start with